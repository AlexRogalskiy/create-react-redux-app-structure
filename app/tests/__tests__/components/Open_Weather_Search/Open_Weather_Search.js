import setupTests from './../../configurations/setupTests';
import React from 'react';
import renderer from 'react-test-renderer';
import Open_Weather_Search from './../../../../components/Open_Weather_Search/Open_Weather_Search';

describe('Open_Weather_Search', () => {
    it('should match snapshot', () => {
        const component = renderer.create(<Open_Weather_Search />);
        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });

    it('should have className app-open-weather-search', () => {
        const wrapper = global.shallow(<Open_Weather_Search />);

        expect(wrapper.find('.app-open-weather-search').hasClass('app-open-weather-search')).toBe(true);
    });
});

