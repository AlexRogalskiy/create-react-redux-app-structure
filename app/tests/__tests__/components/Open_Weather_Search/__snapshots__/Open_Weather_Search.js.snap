// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Open_Weather_Search should match snapshot 1`] = `
<div
  className="app-open-weather-search"
>
  <h3>
    Current weather in your city
  </h3>
  <input
    name="cityName"
    onChange={[Function]}
    onKeyUp={[Function]}
    placeholder="Your city name"
    type="text"
    value=""
  />
  <label>
    <input
      name="isCityList"
      onChange={[Function]}
      type="checkbox"
      value={false}
    />
    <span>
      List of cities
    </span>
  </label>
  <button
    onClick={[Function]}
    type="button"
  >
    Search
  </button>
  <img
    className="app-preload displayNone"
    src="./images/loader.gif"
  />
</div>
`;
